### DISCUSSION TITLE
[Feature Request] Wyze Vaccum Implementation

### DISCUSSION THREAD

**Discussion by tusculumgolfer** [SecKatie/ha-wyzeapi#106] [General]
@JoshuaMulliken, really appreciate all the work you've done on this integration. It's worked really well for me thus far. 

I received two Wyze vacuums last week and they work pretty well thus far. Do you think there is potential to integrate them into your integration? I'm willing to help how I can, but I don't know any programming.

**Comment by SecKatie** [comment]
See this discussion thread for the information that I need to start working on this: https://github.com/JoshuaMulliken/ha-wyzeapi/discussions/103

**Comment by LucasPMagno** [comment]
Did you see my comment in #107? It should have all of the information you need, but let me know if something is missing.
